Documentation update (#979)
ROCmSoftwarePlatform/MITunaX: tuna/miopen/doc/FinDocs.md

> FIN Documentation

Author: ```suggestion
MIFin Documentation
```

==================

> How to run FIN inside a docker!

Author: ```suggestion
How to run MIFin inside a docker!
```

-------------------------------

> To run Fin steps in tuna, a new docker needs to be created, that contains FIN, MIOpen

Author: ```suggestion
To run MIFin steps in MITuna, a new docker needs to be created, that contains MIFin, MIOpen
```
